src/dialogs/ImageOverlay.cpp

```diff
@@ -31,6 +31,8 @@ ImageOverlay::ImageOverlay(QPixmap image, QWidget *parent)
 
         connect(this, SIGNAL(closing()), this, SLOT(close()));
 
+        close_shortcut_ = new QShortcut(QKeySequence(Qt::Key_Escape), this, SLOT(close()));
```

Comment:
It probably makes sense to connect Key_Escape to 'closing' instead of 'close.'  Right now, we don't do any other work after the closing signal is raised, but we could in the future.